server/apps/archive/archive_crop.py

```diff
@@ -88,6 +88,11 @@ def _validate_aspect_ratio(self, crop, doc):
         """
         width = doc['CropRight'] - doc['CropLeft']
         height = doc['CropBottom'] - doc['CropTop']
+
+        if width < crop['width'] or height < crop['height']:
+            raise SuperdeskApiError.badRequestError(
+                message='Wrong crop size. Minimum crop size is {}x{}.'.format(crop['width'], crop['height']))
```

Comment:
If you prefer, you can shorten the error message string with the following (the line appears a bit lengthy):
```python
'Wrong crop size. Minimum crop size is {width}x{height}.'.format(**crop)
```